templates/zerver/app/lightbox_overlay.html

```diff
@@ -1,4 +1,4 @@
-<div id="lightbox_overlay" class="overlay new-style" data-overlay="lightbox">
+<div id="lightbox_overlay" class="overlay new-style" data-overlay="lightbox" aria-hidden="true">
```

Comment:
What's the story with this change?  The lightbox overlay is probably useless to most users who need screen readers (though I suppose it could have some value for folks who are only nearly blind and would still appreciate an image).  But I'm not sure hiding the overlay this way is going to do the right thing; what was your intent here?